php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -615,21 +615,24 @@ class NDB_Menu_Filter extends NDB_Menu
 
         $pageID = isset($_GET['pageID']) ? $_GET['pageID'] : 1;
 
-        $this->tpl_data['rowsPerPage'] = $rowsPerPage;
-        $this->tpl_data['pageID']      = $pageID;
+        $this->setTemplateVar('rowsPerPage', $rowsPerPage);
+        $this->setTemplateVar('pageID', $pageID);
 
-        $this->tpl_data['filterfield'] = isset($_GET['filter']['order']['field'])
-              ? $_GET['filter']['order']['field']
-              : '';
-
-        $this->tpl_data['filterfieldOrder']
-            = isset($_GET['filter']['order']['fieldOrder'])
-              ? $_GET['filter']['order']['fieldOrder']
-              : '';
+        $this->setTemplateVar(
+            'filterfield',
+            $_GET['filter']['order']['field'] ?? ''
```

Comment:
@driusan 

This pull request, now that is merged, is now giving me 500 error on all pages:
`[Thu Jun 15 09:09:59.226826 2017] [:error] [pid 24861] [client 132.216.56.241:48312] PHP Parse error:  syntax error, unexpected '?' in /var/www/loris/php/libraries/NDB_Menu_Filter.class.inc on line 623`